package.json

```diff
@@ -13,8 +13,10 @@
   "scripts": {
     "build": "tsc",
     "build:serverless": "ncc build src/index.ts -o dist",
+    "build:ci": "ncc build adapters/index.ts -o dist && mv ./dist/index.js ./bin",
```

Comment:
We should convert everything fully into typescript.

Reply:
removed both `build:ci` and `build:serverless`